src/anbox/qemu/sensors_message_processor.h

```diff
@@ -18,23 +18,32 @@
 #ifndef ANBOX_QEMU_SENSORS_MESSAGE_PROCESSOR_H_
 #define ANBOX_QEMU_SENSORS_MESSAGE_PROCESSOR_H_
 
+#include "anbox/application/sensors_state.h"
 #include "anbox/qemu/qemud_message_processor.h"
 
 namespace anbox {
 namespace qemu {
 class SensorsMessageProcessor : public QemudMessageProcessor {
  public:
   SensorsMessageProcessor(
-      const std::shared_ptr<network::SocketMessenger> &messenger);
+      std::shared_ptr<network::SocketMessenger> messenger, std::shared_ptr<application::SensorsState> sensorsState);
   ~SensorsMessageProcessor();
 
  protected:
-  void handle_command(const std::string &command) override;
+  void handle_command(const std::string& command) override;
 
  private:
-  void list_sensors();
+  void send_message(const std::string& message);
+  std::shared_ptr<application::SensorsState> sensorsState_;
+  std::atomic<int> delay = 200;
+  std::atomic<bool> temperature = false;
+  std::atomic<bool> run_thread = true;
+  std::thread thread;
```

Comment:
All variables should have a trailing `_` in their name

Reply:
Done.